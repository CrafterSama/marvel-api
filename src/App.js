import React from 'react'
import { createBrowserHistory } from 'history';
import { BrowserRouter as Router } from 'react-router-dom'
import { StateProvider } from './context'
import PrincipalLayout from './components/Layout/PrincipalLayout'
import './App.css'

const history = createBrowserHistory();

function App() {
  return (
    <StateProvider>
      <Router history={history}>
        <PrincipalLayout />
      </Router>
    </StateProvider>
  );
}

export default App;
