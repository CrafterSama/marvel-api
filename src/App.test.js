import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import App from './App';

describe('Test for <App /> Component', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<App />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
