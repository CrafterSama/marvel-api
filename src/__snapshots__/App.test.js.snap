// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src={
        Object {
          "getCacheKey": [Function],
          "process": [Function],
        }
      }
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
